scripts/update_configs.py

```diff
@@ -86,23 +98,100 @@ def _apply_changes_based_on_config(
         writable_local_file.write('\n'.join(local_lines) + '\n')
 
 
-def _update_configs():
-    """Updates the 'feconf.py' and 'constants.ts' files after doing the
+def check_updates_to_terms_of_service():
+    """Checks if updates are made to terms of service and updates
+    REGISTRATION_PAGE_LAST_UPDATED_UTC in feconf.py if there are updates.
+    """
+    personal_access_token = getpass.getpass(
+        prompt=(
+            'Please provide personal access token for your github ID. '
+            'You can create one at https://github.com/settings/tokens: '))
+
+    if personal_access_token is None:
+        raise Exception(
+            'No personal access token provided, please set up a personal '
+            'access token at https://github.com/settings/tokens and re-run '
+            'the script')
+    g = github.Github(personal_access_token)
+    repo = g.get_organization('oppia').get_repo('oppia')
+
+    common.open_new_tab_in_browser_if_possible(
+        'https://github.com/oppia/oppia/commits/develop/core/'
+        'templates/dev/head/pages/terms-page/terms-page.mainpage.html')
+    python_utils.PRINT(
+        'Is terms of service of changed? Check commits/changes made '
+        'to the file: terms-page.mainpage.html. Enter y/ye/yes if they '
```

Comment:
Shouldn't this be computable automatically? I.e. you can generate a list of the commits that changed this file with links to those commits on github. (Or link to the history page on github for the Terms page.)

Also throw an error if it 404s since that may mean the file has been moved.

Reply:
Just to confirm - The script should automatically open the page with the commits on this file but the confirmation will still be provided by the user, right?

Reply:
Yes, that's right.

Reply:
Ah, I see, OK. Sorry, I misread it and thought that it just linked to the latest file contents (I didn't see the "commits" part in the URL).

The second part of my comment still holds, though -- if the file is moved (and the current location is deleted), can you detect that and error out? That check should probably happen near the start of the process, so that the release coordinator doesn't find out that they need to abort only once they are halfway into the process.

Reply:
Yes, got it. Added a check!